### PR TITLE
Added autoCreateSubjects query param

### PR DESCRIPTION
**Pull request by blockbax**
#### Summary

Follows up on the discussion at https://github.com/TheThingsIndustries/lorawan-stack-docs/pull/674

Update on the docs can be found here: https://github.com/TheThingsIndustries/lorawan-stack-docs/pull/753

#### Changes

- Added autoCreateSubjects as field and use as query param
- Updated the descriptions
- Change base-url to dedicated integration (with more user-friendly The Things Stack payload support)

#### Notes for Reviewers

...

#### Checklist

- [x] Scope: The referenced issue is addressed, there are no unrelated changes.
- [x] Compatibility: The changes are backwards compatible, they don't break existing deployments.
- [x] Testing: The changes are tested.
- [x] Documentation: Relevant documentation is added or updated.

### CODE REVIEW

**Comment by nejraselimovic** [comment]
@blockbax I would recommend to leave old field descriptions, these are a bit too long to read. We have a documentation page for this stuff, so no need to make the template packed with text

**Comment by blockbax** [comment]
> I would recommend to leave old field descriptions, these are a bit too long to read. We have a documentation page for this stuff, so no need to make the template packed with text

@nejraselimovic Agree, I removed them completely now, the old were just repeating the names and didn't add much in my opinion.

**Comment by blockbax** [comment]
@nejraselimovic I also updated the docs accordingly, see here: https://github.com/TheThingsIndustries/lorawan-stack-docs/pull/753

File: blockbax.yml [hunk 1]
```diff
@@ -6,18 +6,20 @@ info-url: https://blockbax.com/
...
+    name: Automatically create subjects?
+    secret: false
+    default-value: "true"
 format: json
 headers:
   Authorization: ApiKey {token}
   Content-Type: application/json
 create-downlink-api-key: false
-base-url: https://api.blockbax.com/v1/projects/{projectId}/measurements
+base-url: https://api.blockbax.com/v1/projects/{projectId}/integrations/53c06128-ce26-4dcc-8d74-5c45b302b525/measurements?autoCreateSubjects={autoCreateSubjects}
```
Comment (adriansmares) [inline blockbax.yml:23]: Is it intentional that the `53c06128-ce26-4dcc-8d74-5c45b302b525` is hardcoded here ? Is it not `{projectId}` ?

Comment (blockbax) [reply blockbax.yml:23]: Yes it is intentional, we set this integration up to better accommodate the default webhook payload coming from The Things Stack, making it more user-friendly. Any project can use this. Thanks for checking!

**Review by nejraselimovic — APPROVED** [review]

**Pull request merged**